tf_agents/cogment_verse_tf_agents/reinforce/reinforce.py

```diff
@@ -126,19 +120,27 @@ def sample_training_batch(self, batch_size=32):
 
         return rval
 
-    def reset_replay_buffer(self):
+    def _reset_replay_buffer(self):
         """
         Resets buffer
         """
         self._replay_buffer._write_index = -1
         self._replay_buffer._n = 0
```

Comment:
Weird to be accessing private member of the replay buffer like that, what does this function do anyway ?

Reply:
resets the replay buffer

Reply:
Not sure what it means in practice. My point is that you are using internal implementation details of the replay buffer class so it's a little fragile. 

Ideally this should be a function of the replay buffer, if it doesn't exist, in this case I wouldn't add it directly because it's third party code. But at least make sure that it really explains what it's supposed to do so that a future developer will know what to change if it breaks.